src/tito/release/main.py

```diff
@@ -527,6 +527,7 @@ def _koji_release(self):
                 if scl:
                     builder = copy.copy(self.builder)
                     builder.scl = scl
+                    builder.cleanup()
```

Comment:
I'm unsure if calling cleanup could leave the *original* builder (self.builder) in a strange state if it has already been used?  (The source dir etc might have been cleaned, but self.builder thinks it's dirty still.)